platform/innovium/sonic-platform-modules-wistron/6512-32r/sonic_platform/chassis.py

```diff
@@ -141,15 +151,28 @@ def get_reboot_cause(self):
         return (reboot_cause, description)
 
     def _get_sku_name(self):
-        p = subprocess.Popen(GET_HWSKU_CMD, shell=True, stdout=subprocess.PIPE)
+        p = subprocess.Popen(GET_HWSKU_CMD, stdout=subprocess.PIPE)
```

Comment:
@maipbui , the code related with PR#[12102](https://github.com/sonic-net/sonic-buildimage/pull/12102) at here.

there are many modifications like this, I only pointed one here.

Reply:
we moved the code from device folder to platform folder.